### ISSUE TITLE
Simplify templates hierarchy

### ISSUE THREAD

**Issue by chrismedrela** [swcarpentry/amy#918]
Now we have five base templates:
- `base_nonav_fixed.html`
- `base_nav_fixed.html`
- `base_nav_fluid.html`
- `base_nav_fluid_sidebar.html`
- `base_nav_fluid_twocolumn.html`

I'd be simpler if we had only the last three templates, especially that such complexity results in some bugs:
- The distinction between `fixed` and `fluid` results in bugs like #854.
  
  We could stick with `fluid` and get rid of `fixed`. What's more, we wouldn't have both `navigation_fixed.html` and `navigation_fluid.html`.
- `nonav` vs `nav` results in bugs like #856.
  
  We could get rid of `nonav` and stick with `nav`. What's more, we wouldn't have both `generic_form.html` and `generic_form_nonav.html`. The navigation would be displayed only if the current user is admin, so we need something like:

```
{% if user.is_admin %}
  {% include "navigation.html" %}
{% endif %}
```

@pbanaszkiewicz, any opinion?

**Comment by gvwilson** [comment]
+1, but let's leave this for end-of-summer cleanup - we need to polish 
off instructor training registration workflow first.

**Comment by pbanaszkiewicz** [comment]
> I'd be simpler if we had only the last three templates, especially that such complexity results in some bugs:

The distinction for wider/narrower pages was made long ago, and as such you had inherited it -- but it was okay at the time; clearly with newer developments of PyData/training workflow the design is a little outdated.

> +1, but let's leave this for end-of-summer cleanup

I agree, but I'd really like to move the external forms out of core `workshops` application (or at least its URL scope) first.

**Comment by chrismedrela** [comment]
@pbanaszkiewicz, what are "external forms"?

**Comment by pbanaszkiewicz** [comment]
@chrismedrela see #816.

**Comment by pbanaszkiewicz** [comment]
Closed in #1311.
